clap_complete_nushell/tests/common.rs

```diff
@@ -23,6 +23,17 @@ pub fn basic_command(name: &'static str) -> Command {
         )
 }
 
+pub fn multiline_command(name: &'static str) -> Command {
+    Command::new(name)
+        .about("This is a long about\nwhich goes over 2 lines and ends with a dot.")
+        .arg(
+            Arg::new("test")
+                .action(ArgAction::SetTrue)
+                .long("test")
+                .help("This is a long description\nwhich goes over 2 lines and ends with a dot."),
+        )
+}
```

Comment:
We are in-progress switching out test styles.  These older tests only show what the generated script will be.  The newer tests show *and verify* them.

If you could
- Move this fixture into `clap_complete_nushell/examples/test.rs` and verify it in `clap_complete_nushell/tests/completion.rs`
  - Add it to the `quote` command as that is where we put all things escaping related
- Can you mirror the fixture in `clap_complete/examples/exhaustive.rs` though no verification is needed to be added

Reply:
I tried to do that, but i have 2 problems
1. it seems that the version of nushell which completest uses is too old,
it doesn't behave as my local installation of nushell (0.85.0)

test:
```
thread 'completion_multi_line' panicked at 'Eq failed:
--- Expected
+++ Actual
   1    1 | % test quote --multi-line
   2      - --multi-line    This is a long description which goes over 2 lines and ends with a dot.
        2 + --multi-line    This is a long description
        3 + which goes over 2 lines and ends with a dot.

', clap_complete_nushell/tests/completion.rs:97:5
```

local shell:
[![asciicast](https://asciinema.org/a/DN7drmMfvGvHKis69BChIMyF3.svg)](https://asciinema.org/a/DN7drmMfvGvHKis69BChIMyF3)

2. I'm not sure how i should write a test to show the buggy behaviour as mentioned in [this comment](https://github.com/clap-rs/clap/pull/5165#discussion_r1350511781), because the generated code isn't valid nu code

Reply:
> it seems that the version of nushell which completest uses is too old,

I've tried doing an update but I don't know enough nushell. Contributions welcome!

> I'm not sure how i should write a test to show the buggy behaviour as mentioned in https://github.com/clap-rs/clap/pull/5165#discussion_r1350511781, because the generated code isn't valid nu code

When adding the case, the test will instead assert that an error is returned.

Reply:
> When adding the case, the test will instead assert that an error is returned.

This would also break other tests, when I add it to the quote command, because of the invalid code. How should I handle that?

Reply:
Other nushell tests?  Thats fine.